django_stubs_ext/tests/test_monkeypatching.py

```diff
@@ -46,6 +47,9 @@ def test_patched_generics(make_generic_classes: _MakeGenericClasses) -> None:
         if el.version is None:
             assert el.cls[type] is el.cls  # `type` is arbitrary
 
+    class TestForm(forms.ModelForm[type]):
```

Comment:
You need to: 
1. use an actual model type here (if it is already created)
2. just add a `# type: ignore`, because we test a different thing

Reply:
👍 All fixed, no need for the `# type: ignore`.